src/encryption/encryption.service.ts

```diff
@@ -0,0 +1,38 @@
+import { Injectable } from '@nestjs/common'
+import { KmsKeyringNode, encrypt, decrypt } from '@aws-crypto/client-node'
+import { ConfigService } from '@nestjs/config'
+import { IKMSConfig } from '../config/kmsConfig'
+
+@Injectable()
+export class EncryptionService {
+  private readonly kmsKeyRing: KmsKeyringNode
+
+  constructor(private configService: ConfigService) {
+    const config = configService.get<IKMSConfig>('kms')
+    if (!config) {
+      throw new Error('Missing KMS config')
+    }
+    this.kmsKeyRing = new KmsKeyringNode({ generatorKeyId: config.generateKeyId })
+  }
+
+  async encrypt(plaintext: string, encryptionContext: Record<string, string>): Promise<any> {
```

Comment:
I think this return type should be `Promise<Buffer>` and the field type of `PlaidItem.accessToken` should also be Buffer. I haven't tested it locally because I don't have plaid config setup but I think that should work and avoid `any` usage.

Reply:
Good catch, updated both to Buffer and verified encryption/decryption still works.